src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.spec.ts

```diff
@@ -64,7 +64,7 @@ describe('DotEditPageNavComponent', () => {
             component = fixture.componentInstance;
         });
 
-        it('should have menu items: Content and Code', () => {
+        it('should have menu items: Content only', () => {
```

Comment:
Why?

Reply:
we should hide the <code> icon, so the user is not able to edit the advance template

Reply:
Ok but that means you need to create a new tests for that scenario.